plugins/woocommerce-admin/client/products/fields/attribute-field/add-attribute-modal.scss

```diff
@@ -54,10 +60,37 @@
 		td:not(:last-child) {
 			margin-right: $gap;
 		}
+
+		@include breakpoint('<782px') {
+			position: relative;
+			grid-template-columns: 1fr;
+			padding-right: 42px;
```

Comment:
Would it be worth adding a comment here, mentioning this padding is for the `trash-column` to display correctly?